amd64-usbtpu.Dockerfile

```diff
@@ -94,6 +95,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 ENV DEV_ALLOW_ALL_ORIGINS=true
 ENV AWS_SHARED_CREDENTIALS_FILE=/repo/.aws/credentials
 ENV AWS_CONFIG_FILE=/repo/.aws/config
+ENV CONFIG_FILE=config-coral.ini
```

Comment:
If you want to use another config file you would now have to rebuild the steps behind this so I would prefer to have it at the end

Reply:
You are right. I changed it.